source/testnet/see-also/release-notes.rst

```diff
@@ -17,6 +14,38 @@ Release Notes
    :local:
    :backlinks: none
 
+Open Testnet v4
+===============
+
+January 13th, 2020
+
+Smart contracts:
+
+* Smart contracts support on chain
+* Rust supported as off-chain SC language
+* Cargo-concordium tool for building and testing smart contracts off-chain
+* Documentation for smart contracts added to `developer docs <https://concordium.github.io/en/testnet4/smart-contracts/index.html>`_
+* SC transactions added to Concordium client
+
+Tokenomics (to match tokenomics model):
+
+* Rewards for baking and finalization changed
+* Minting changed
+* Chain parameters updated
+* Updated `network dashboard block explorer <https://dashboard.testnet.concordium.com/chain>`_ to include new info
+* Amount lock-up transaction with schedule added
+* Staking changed so staked amount is locked
+* Mobile app updated to show staking and amount lockup schedules
+* Delegation disabled
+
+ID provider:
```

Comment:
Should this be "ID layer" perhaps?

Reply:
Sure! @smh1001, also cool with you to change this?

Reply:
This is now called ID Layer